### PR TITLE
fix: Update code coverage step

### PR DESCRIPTION
**Pull request by NotMyFault**
Fixes https://github.com/jenkins-infra/jenkins-io-components/issues/84

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -102,7 +102,7 @@ pipeline {
       }
       post {
         always {
-          publishCoverage adapters: [cobertura('coverage/storybook/cobertura-coverage.xml')], checksName: 'coverage', sourceFileResolver: sourceFiles('NEVER_STORE')
+          recordCoverage name: 'coverage', sourceCodeRetention: 'NEVER', tools: [[parser: 'COBERTURA', pattern: 'coverage/storybook/cobertura-coverage.xml']]
```
Comment (halkeye) [inline Jenkinsfile:105]: how does sourcecode retention of never work when there's no workspaces kept around? will it just report totals or something then?

....and I see it was essentially that in the past, so its probably fine.

**Comment by halkeye** [comment]
yolo

**Comment by halkeye** [comment]
oops, this should have been chore: or ci: since it didn't need a release

File: Jenkinsfile (hunk 1 above)
Comment (NotMyFault) [reply Jenkinsfile:105]: There shouldn't be a change to the previous layout, given files weren't stored before, either.

**Pull request merged**